src/m365/spfx/commands/project/project-upgrade/upgrade-1.15.0.ts

```diff
@@ -66,6 +67,7 @@ module.exports = [
   new FN002009_DEVDEP_microsoft_sp_tslint_rules('', false),
   new FN002013_DEVDEP_types_webpack_env('1.15.2'),
   new FN002018_DEVDEP_microsoft_rush_stack_compiler_3_9('', false),
+  new FN002019_DEVDEP_spfx_fast_serve_helpers('1.15.0'),
```

Comment:
Good catch. I will update it

Reply:
Thanks for the awesome-fast turnaround 👏

Reply:
Sorry, maybe I am a bit later here, but could we actually use `~1.15.0` in the upgrade suggestion? It's recommended pattern since I publish fixes to patch versions, so with tilde `~` on a fresh install, you always have the latest patch version with all possible fixes. 
For example, the current version is  already `1.15.3`, having `~1.15.0` will install `1.15.3` in that case.

Reply:
Totally agree with that idea. This is something which we briefly discussed in the first PR that `spfx-fast-serve-helpers` was introduced within the command `spfx project upgrade`.

- https://github.com/pnp/cli-microsoft365/pull/3161#discussion_r833156329

To implement this we need to adjust the current version comparison logic. But before we go to in depth on this idea. I will open a new issue so we can have a deeper discussion over there. 😄